### PR TITLE
Better Fullbright

### PR DESCRIPTION
**Pull request by N0tExisting**
Improved Fullbright by reseting gamma to the vaule before activation.

### CODE REVIEW

File: Horion/Module/Modules/FullBright.cpp [hunk 1]
```diff
@@ -10,21 +10,25 @@ const char* FullBright::getModuleName() {
 	return "Fullbright";
 }
 
-void FullBright::onTick(C_GameMode* gm) {
-	if (gammaPtr != nullptr && *gammaPtr != 10) 
+void FullBright::onEnable() {
+	if (gammaPtr != nullptr) {
```
Comment (totally-not-important) [inline Horion/Module/Modules/FullBright.cpp:14]: unidentified

File: Horion/Module/Modules/FullBright.cpp [hunk 2]
```diff
@@ -10,21 +10,25 @@ const char* FullBright::getModuleName() {
...
+		prevGamma = *gammaPtr;
```
Comment (ghost) [inline Horion/Module/Modules/FullBright.cpp:15]: everything under prevGamma is unidentified or undeclared

Comment (N0tExisting) [reply Horion/Module/Modules/FullBright.cpp:15]: What do you mean?
gammaPtr gets defined in Hooks.cpp([1349](https://github.com/horionclient/Horion/blob/master/Memory/Hooks.cpp#L1349)) and i use a negative value in prevGamma to say that i don't know the users gamma yet

Comment (ghost) [reply Horion/Module/Modules/FullBright.cpp:15]: I was stupid as fuck and didn't add the .h file sorry.

**Comment by GodGamer029** [comment]
I'll add that myself, good idea!

**Comment by N0tExisting** [comment]
> I'll add that myself, good idea!

sure ... =_=

**Pull request closed without merging**